### ISSUE TITLE
Copy from playback history

### ISSUE THREAD

**Issue by MuntashirAkon** [iina/iina#2376]
**System and IINA version:**

- IINA 1.0.2

**Expected behavior:**

When pressing <tt>Cmd</tt> + <tt>C</tt> on an active row in the playback history, the item (URL or Directory) should be copied and/or right click on an item in the playback history there should be an option for copying the item.

This is especially useful if I play from a URL as I may need to copy it for other purposes (e.g. URL sharing).

**Actual behavior:**

Currently, I have to manually copy from `~/Library/Application\ Support/com.colliderli.iina/history.plist`

**Comment by anohren** [comment]
Yes, I've been meaning to suggest the same context menu options for the history window as those suggested in https://github.com/iina/iina/issues/2271
